.travis.yml

```diff
@@ -15,7 +16,11 @@ matrix:
         skip_cleanup: true
         on:
           branch: master
+
+before_script:
+  - rustup component add rustfmt-preview
 script:
+  - [ "$RUST_STABLE" = 1 ] && cargo fmt --all -- --check || test 1
```

Comment:
`test 1` is a total bashism. I'd rather keep to the Unix tradition of no news is good news.